src/main/java/codeu/model/data/Tag.java

```diff
@@ -27,6 +30,11 @@ public UUID getId() {
     return id;
   }
 
+  /** Returns the ID of the Conversation this Tag belongs to. */
+  public UUID getConversationId() {
```

Comment:
Can't a tag belong to more than one conversation? So if I tag multiple conversations with the word "work", then there should be only one tag object that stores references to each conversation. 
So perhaps we should instead have a list of conversation Ids